src/main/java/edu/ucsb/ucsbcslas/controllers/TutorController.java

```diff
@@ -81,7 +81,8 @@
     }
 
     @PostMapping(value = "/api/member/tutors/upload", produces = "application/json")
-    public ResponseEntity<String> uploadCSV(@RequestParam("csv") MultipartFile csv, @RequestHeader("Authorization") String authorization) throws IOException {
+    public ResponseEntity<String> uploadCSV(@RequestParam("csv") MultipartFile csv,
+            @RequestHeader("Authorization") String authorization) throws IOException {
         logger.info("Starting upload CSV");
         AppUser user = authControllerAdvice.getUser(authorization);
```

Comment:
We don't do anything with `user` we should check whether the current user is authorized to make this POST request.

Reply:
I did not work on the PR for uploading CSV's for the Tutor. I think the lines got modified when I auto formatted this file

Reply:
This is something that should definitely be noted in an issue, but I agree doesn't pertain directly to this PR